cli/fathom_web/commands/histogram.py

```diff
@@ -0,0 +1,80 @@
+from json import load
+
+from click import argument, command, File, option
+from plotly.graph_objects import Histogram
+from plotly.subplots import make_subplots
+
+
+@command()
+@argument('vector_file', type=File('r'))
+@option('--feature_name', '-f',
+        default='',
+        help='The name of the feature to graph. Omit to graph all features.')
```

Comment:
If you can only pass one `vectors.json` file, is it possible to have more than one feature?

Reply:
Ah by feature here we mean a "rule" in Fathom; I keep periodically confusing feature and class (or "type" in Fathom). So I guess I could graph just one feature (e.g. `hasLongTextContent` in my paragraph ruleset).

Reply:
I should say "type" for consistency.